\documentclass[../notes.tex]{subfiles}

\pagestyle{main}
\renewcommand{\chaptermark}[1]{\markboth{\chaptername\ \thechapter\ (#1)}{}}
\setcounter{chapter}{5}

\begin{document}




\chapter{Import and Export}
\section{Organelles and Transport}
\begin{itemize}
    \item \marginnote{11/1:}Warm-up activity: Quiz questions.
    \item This class and next class: Protein localization mechanisms and inhibition.
    \item Think about\dots
    \begin{itemize}
        \item How a cell transports and localizes proteins;
        \item Mechanisms of preventing things from going where they should.
    \end{itemize}
    \item Indeed, some protein inhibitors work not by inactivating proteins but by making sure they don't get to the right place.
    \item \textbf{Chaperone}: A small molecule that allows a misfolded protein in the wrong place to fold and reach its site of action.
    \begin{itemize}
        \item These are promising new drugs.
        \item Example: There is a known risk gene for Parkinson's disease. A protein gets stuck in the ER. If there are small molecules you can use to get the protein to fold in the ER and be released, you win.
        \item Example: Cardiovascular disease. Most drugs fail clinical trials right at the last stage of testing because they cause something called \textbf{long QT syndrome}.
        \begin{itemize}
            \item Said drugs cause this syndrome by preventing ion channels from reaching the plasma membrane.
            \item Chaperones could potentially help overcome this common barrier.
        \end{itemize}
    \end{itemize}
    \item \textbf{Arrhythmia}: A fast, chaotic heartbeat.
    \item \textbf{Long QT syndrome}: A heart signaling disorder that can cause arrhythmias.
    \begin{itemize}
        \item Symptoms can be severe, up to death.
    \end{itemize}
    \item Today: Mechanisms of protein localization.
    \begin{itemize}
        \item How do proteins reach the nucleus, mitochondria, and a mystery organelle? These are open questions in basic biology.
    \end{itemize}
    \item Organelles and membranes by the numbers.
    \begin{itemize}
        \item Cytosol: 2\% of the membrane in a cell, but 54\% of total cell volume.
        \item Thus, the plasma membrane spends a lot of energy keeping the cytosol happy.
        \item The mitochondria and ER have a huge amount of membrane but very little volume and contribute to keeping the cytosol happy as well.
        \begin{itemize}
            \item The membrane content helps maintain homeostasis.
        \end{itemize}
        \item What was the net point of all this??
    \end{itemize}
    \item Evolution of compartments.
    \begin{itemize}
        \item Helps us understand \textbf{topological equivalence}.
        \item Yamuna believes that the endosymbiotic theory is just a hypothesis and that it's all up in the air and likely to change.
        \item Current hypothesis: Archaea lost its cell wall making it easier for it to acquire DNA. Once it acquired enough valuable genes, the cell membrane underwent an invagination to form the nucleus and extra folds of the ER. This prevents the cell from losing the DNA its acquired.
        \begin{itemize}
            \item This is why the ER and extracellular matrix are topologically equivalent, i.e., because the former evolved from the latter.
        \end{itemize}
        \item Mitochondria are the cell intaking another bacteria that could produce energy.
    \end{itemize}
    \item \textbf{Topologically equivalent} (compartments): Two compartments inside (or outside) a cell such that materials do not have to cross a membrane to get from one to the other.
    \begin{figure}[h!]
        \centering
        \includegraphics[width=0.4\linewidth]{../ExtFiles/topologicalEquivalence.png}
        \caption{Topological equivalence.}
        \label{fig:topologicalEquivalence}
    \end{figure}
    \begin{itemize}
        \item Example: Golgi and ER are topologically equivalent (and topologically equivalent to the extracellular matrix) but not to the cytoplasm. This is because materials in the ER move to the Golgi and then to the extracellular matrix within a \textbf{vesicle}, i.e., they never have to cross a plasma membrane so much as they get surrounded and moved by different membranes.
        \item Example: The extracellular matrix and the cytoplasm are not topologically equivalent. Notice that any material coming into the cytoplasm from the outside must cross through the plasma membrane using one of the mechanisms from last class (we're not talking endocytosis yet).
    \end{itemize}
    \item Proteins can be transported between organelles either by being stuck in the membrane of a vesicle (at which point they will end up in the membrane of the target organelle) or within said vesicle's lumen (at which point they will end up in the lumen of the target organelle).
    \item Isolating organelles.
    \begin{figure}[h!]
        \centering
        \includegraphics[width=0.5\linewidth]{../ExtFiles/isolatingOrganelles.jpeg}
        \caption{Isolating organelles.}
        \label{fig:isolatingOrganelles}
    \end{figure}
    \begin{itemize}
        \item We discover transport mechanisms by carrying out a lot of mutations and then isolating specific target organelles and testing for a protein's presence (look for a ratio between the quantity of this protein present and a standard protein that you know will be there).
        \item Isolation mechanism: We take a bunch of cells, dissolve the extracellular membrane to get a soup of organelles, centrifuge it (to let heavy organelles like the nucleus fall out), centrifuge again (to get the small organelles like the mitochondria, lysosomes, and peroxisomes), centrifuge it again (to get fragments of the plasma membrane and ER), and centrifuge it one last time (to get ribosomes).
        \item Alternative isolation mechanism: Use a matrix with a density gradient and just centrifuge once to get multiple layers.
    \end{itemize}
    \item Three main ways to move proteins in a cell.
    \begin{itemize}
        \item Recall passive and active exchange.
        \item You can let physical equilibrium take hold, of course, but often that won't lead to great enough concentrations.
        \item Thus, cells evolved the following three methods\dots
    \end{itemize}
    \item \textbf{Gated transport}: A type of transport involving a gate and a condition (e.g., a binding or membrane potential) that must be satisfied for the gate to "lift open."
    \item \textbf{Translocation}: The movement between topologically nonequivalent compartment.
    \begin{itemize}
        \item Example: Suppose you have a protein that's been made in the ER, has been deposited into the cytoplasm, and now needs to get into the mitochondria. We will consider this example in much greater detail shortly.
    \end{itemize}
    \item \textbf{Vesicular transport}: The movement of biomolecules in vesicles between topologically equivalent compartments.
    \item \textbf{Localization sequence}: A molecular GPS. \emph{Also known as} \textbf{nuclear localization sequence}, \textbf{NLS}.
    \begin{itemize}
        \item Usually located on the N-terminus because it comes out first and needs to know where to go.
        \item Localization sequences have different strengths. Some send proteins in a high fraction somewhere; some send proteins in a low fraction somewhere.
        \begin{itemize}
            \item Strength is determined by the sequence's affinity for the transport protein. We will discuss this in more depth later.
        \end{itemize}
        \item Length: Tetrapeptides up to 20-30 AAs.
        \item Very occasionally occur in the middle of a protein.
    \end{itemize}
    \item \textbf{Translocation sequence}: A molecular GPS on the C-terminus that moves a protein after folding.
    \item Gated transport example: Movement from the cytosol into the nucleus.
    \begin{itemize}
        \item This is also the most common type of gated transport.
        \item The gate is the nuclear pore, and the condition is \textbf{karyopherin} binding
    \end{itemize}
    \item \textbf{Karyopherin}: A protein involved in transporting molecules between the cytoplasm and the nucleus.
    \item Consider first the structure of the \textbf{nuclear pores}.
    \item \textbf{Nuclear pore}: A gateway from the cytosol to the nucleus.
    \begin{figure}[h!]
        \centering
        \includegraphics[width=0.5\linewidth]{../ExtFiles/nuclearPore.png}
        \caption{Nuclear pore structure.}
        \label{fig:nuclearPore}
    \end{figure}
    \begin{itemize}
        \item Nuclei have double plasma membranes and nuclear pores. All transport in and out of the nucleus occurs via nuclear pores.
        \item \textbf{Membrane ring proteins} make the membrane bend backward around nuclear pores.
        \item There are about 3000 nuclear pores per nucleus.
        \item About 1000 molecules transport both ways per nuclear pore per second.
        \item Active v. passive transport: Anything smaller than \SI{40}{\nano\meter} will freely diffuse to a significant extent (smaller implies higher passive transport). Larger, you need something to capture it and drag it through (this is active transport).
        \item Nuclear pores are 8-fold symmetric bodies.
        \begin{itemize}
            \item We still don't know the complete structure.
            \item Composed of \textbf{nucleoporins}.
            \item Hair-like \textbf{cytosolic fibrils} on the outside and a \textbf{nuclear basket} on the inside.
            \item A porous plug in the center; still don't know what it is, but it's made of lots of FG repeats.
        \end{itemize}
    \end{itemize}
    \item \textbf{Nucleoporin}: A protein that is a constituent building block of the nuclear pore complex. \emph{Also known as} \textbf{nap}.
    \begin{itemize}
        \item There are permanent naps, but there are also naps which come off and on.
        \item Approximately 30 exist.
        \item Some are transmembrane.
    \end{itemize}
    \item Probing NLS-enabled nuclear import.
    \begin{figure}[H]
        \centering
        \includegraphics[width=0.5\linewidth]{../ExtFiles/NLSimport.png}
        \caption{Nuclear import receptor binding.}
        \label{fig:NLSimport}
    \end{figure}
    \begin{itemize}
        \item Suppose we fuse an NLS with GFP (for which a Nobel Prize has been awarded).
        \item If, in the NLS sequence, we change a $\text{K}\to\text{T}$, then localization of the sequence is significantly decreased.
        \begin{itemize}
            \item Review: Replacing positively charged lysine with polar threonine would certainly affect the interaction between the NLS and the \textbf{nuclear import receptor}!
        \end{itemize}
        \item Conclusion: We can affect NLS efficiency by altering one's affinity for its karyopherin (or vice versa), or by altering the ability of the karyopherin to enter the nucleus.
        \item "It depends upon which bus you get on and upon the strength of your ticket."
        \item Benefit of differential binding affinities: It is possible to have different concentrations of different proteins. You don't want all proteins in the nucleus to have the same concentration, after all.
        \item There also exist \textbf{nuclear import adaptor proteins} which link cargo proteins to their nuclear import receptors with higher binding affinities.
    \end{itemize}
    \item Nuclear export is the reverse of nuclear import.
    \begin{itemize}
        \item Before we can discuss nuclear export directly, we should discuss the Ran proteins.
        \begin{figure}[h!]
            \centering
            \begin{subfigure}[b]{0.49\linewidth}
                \centering
                \includegraphics[width=0.8\linewidth]{../ExtFiles/nuclearImpExpa.png}
                \caption{The Ran proteins.}
                \label{fig:nuclearImpExpa}
            \end{subfigure}
            \caption{Nuclear import and export mechanism.}
        \end{figure}
        \begin{itemize}
            \item Ran complexes have a domain called a \textbf{GTPase domain}.
            \item Ran's GTPase domain has GTP- and GDP-bound forms.
            \item There is a Ran-GDP / Ran-GTP gradient across the nuclear membrane: Ran-GTP is present in much higher concentrations within the nucleus, and Ran-GDP is present in much higher concentrations outside the nucleus.
            \item Ran-GAP is a \textbf{GAP} for Ran-GTP and Ran-GEF is a \textbf{GEF} for Ran-GDP.
            \item Ran-GAP is localized in the cytosol, and Ran-GEF is localized in the nucleus (it sits on chromatin inside the nucleus).
        \end{itemize}
        \item We are now ready to discuss nuclear import and export.
        \begin{figure}[h!]
            \ContinuedFloat
            \centering
            \begin{subfigure}[b]{0.49\linewidth}
                \centering
                \includegraphics[width=0.82\linewidth]{../ExtFiles/nuclearImpExpb.png}
                \caption{Nuclear import.}
                \label{fig:nuclearImpExpb}
            \end{subfigure}
            \begin{subfigure}[b]{0.49\linewidth}
                \centering
                \includegraphics[width=0.7\linewidth]{../ExtFiles/nuclearImpExpc.png}
                \caption{Nuclear export.}
                \label{fig:nuclearImpExpc}
            \end{subfigure}
            \caption{Nuclear import and export mechanism.}
            \label{fig:nuclearImpExp}
        \end{figure}
        \begin{itemize}
            \item Nuclear importers first bind their proteins. Their hydrophobic regions are then caught by the cytosolic fibrils. Moving downward into the FG repeats, the importer's movement once inside is a random walk.
            \item Once an importer arrives in the nucleus, Ran-GTP attacks. Ran-GTP has a higher affinity for it than its substrate, so it will bind and cause the substrate to fall off, completing delivery to the nucleus.
            \item When the Ran-GTP-bound importer diffuses back out of the nucleus, Ran-GAP promotes GDP hydrolysis, and Ran-GDP dissociates.
            \item Nuclear export receptors random walk into the nucleus, bind a Ran-GTP, engage the cargo, random walk out of the nucleus, Ran-GAP hydrolyzes Ran-GTP to RanGDP which leaves, and this kicks out the cargo.
        \end{itemize}
        \item Note that as we would expect for an example of gated transport, a condition is met and only then does transport occur.
    \end{itemize}
    \item \textbf{GTPase domain}: A region of a protein that hydrolyzes a GTP to release energy, accelerating and powering the function of the protein.
    \begin{itemize}
        \item Carried by many kinds of proteins and is very powerful.
        \item Can help a ribozome work, help proteins move from the nucleus to the cytosol, promote vesicle fusing, etc.
        \item Essentially functions as a backpack with a battery.
    \end{itemize}
    \item \textbf{GAP}: A protein that promotes GTP hydrolysis in a GTPase domain that's already bound to GTP. \emph{Also known as} \textbf{GTPase activating protein}.
    \item \textbf{GEF}: A protein that exchanges GDP for GTP at the GTPase domain. \emph{Also known as} \textbf{Guanine nucleotide exchange factor}.
    \item Translocation example: Movement from the cytosol into a mitochondrion.
    \begin{itemize}
        \item Mitochondria have proteins that sit specifically on the outer membrane, inner membrane, in the interluminal space, or in the center of the matrix. This indicates very high accuracy and targeting.
        \begin{itemize}
            \item Many mitochondrial diseases occur due to poor localization.
        \end{itemize}
        \item The lessons here are broadly applicable.
        \item Before next class, brush up on translation.
        \item This is an example of \textbf{post-translational} protein transport.
        \item There is also \textbf{co-translational} protein transport, but we'll talk about that another day.
    \end{itemize}
    \item \textbf{Post-translational} (protein transport): Having a protein cross a membrane after its ribosome has finished synthesizing it.
    \item \textbf{Co-translational} (protein transport): Having a protein cross a membrane as it is still being synthesized by a ribosome.
    \begin{itemize}
        \item Usually happens in the ER.
    \end{itemize}
    \item There are four main mitochondrial proteins/complexes to consider: \textbf{TIM}, \textbf{TOM}, \textbf{SAM}, and \textbf{OXA}.
    \begin{figure}[h!]
        \centering
        \includegraphics[width=0.5\linewidth]{../ExtFiles/MiTranslocators.png}
        \caption{Mitochondrial translocators.}
        \label{fig:MiTranslocators}
    \end{figure}
    \begin{itemize}
        \item These are all big, multiprotein complexes assembled on the various membranes.
    \end{itemize}
    \item \textbf{TOM complex}: The mitochondrial complex of proteins --- localized in the outer membrane --- responsible for the movement of proteins through this barrier and into the interluminal space. \emph{Also known as} \textbf{translocase of the outer membrane}.
    \item \textbf{TIM complex}: The mitochondrial complex of proteins --- localized in the inner membrane --- responsible for the movement of proteins through this barrier and into the matrix. \emph{Also known as} \textbf{translocase of the inner membrane}, \textbf{TIM23}.
    \item \textbf{SAM complex}: The mitochondrial complex of proteins --- localized in the outer membrane --- responsible for the folding/insertion of proteins into the outer membrane. \emph{Also known as} \textbf{sorting and assembly machinery complex}.
    \item \textbf{OXA complex}: The mitochondrial complex of proteins --- localized in the inner membrane --- responsible for the movement of proteins through this barrier and into the matrix. \emph{Also known as} \textbf{oxidase assembly complex}.
    \item There is a way to get TIM and TOM to lock together so translocation happens all at once from the cytosol to the matrix (instead of having to pass through the interluminal space).
    \begin{figure}[H]
        \centering
        \includegraphics[width=0.97\linewidth]{../ExtFiles/MiTransCytMat.png}
        \caption{Translocation from the cytosol to the mitochondrial matrix.}
        \label{fig:MiTransCytMat}
    \end{figure}
    \begin{itemize}
        \item Role of energy in protein import into the mitochondrial matrix.
        \begin{itemize}
            \item Every ATP hydrolyzed at TOM causes you to pull the protein through by a couple of peptides.
            \item Membrane potential drives TIM.
        \end{itemize}
        \item Once the whole protein has been pulled through TOM, TOM and TIM separate.
        \item Once the translocation sequence has completely entered the matrix, a signal peptidase cleaves it, trapping the protein in the matrix.
    \end{itemize}
    \item We now talk about how proteins are sent to each membrane.
    \item Sending proteins to the outer membrane.
    \begin{figure}[h!]
        \centering
        \includegraphics[width=0.35\linewidth]{../ExtFiles/MiTransCytOut.png}
        \caption{Translocation from the cytosol to the mitochondrial outer membrane.}
        \label{fig:MiTransCytOut}
    \end{figure}
    \begin{itemize}
        \item Many porins are present on the outer membrane.
        \item A protein first gets pulled into the interluminal space.
        \begin{itemize}
            \item Aside: In bacteria, this space is known as the \textbf{periplasm}.
            \item The periplasm is very nice for protein generation because there are very few proteins there and once you clone proteins there, all you have to do to release them is crack open the cell wall.
        \end{itemize}
        \item When proteins get pulled into the intermembrane space, they are all hydrophobic (because they will reside in a phospholipid bilayer eventually). Thus, they are prone to agregation in their water-based media, but chaperones latch on to separate them. Once stabilized in the intermembrane space, the protein then gets sent to SAM which folds it into the membrane. SAM has a slit, so as it pulls peptides in, it ejects them out laterally into the membrane.
        \begin{itemize}
            \item Aside: In a bacteria, there is an analogous BAM complex.
            \item Implication: This process is conserved between bacteria and mitochondria, further supporting the endosymbiotic theory.
        \end{itemize}
    \end{itemize}
    \item Sending proteins to the inner membrane.
    \begin{figure}[H]
        \centering
        \begin{subfigure}[b]{0.4\linewidth}
            \centering
            \includegraphics[width=0.8\linewidth]{../ExtFiles/MiTransCytIna.png}
            \caption{Direct insertion.}
            \label{fig:MiTransCytIna}
        \end{subfigure}
        \begin{subfigure}[b]{0.4\linewidth}
            \centering
            \includegraphics[width=0.98\linewidth]{../ExtFiles/MiTransCytInb.png}
            \caption{Via the OXA complex.}
            \label{fig:MiTransCytInb}
        \end{subfigure}\\[1em]
        \begin{subfigure}[b]{0.4\linewidth}
            \centering
            \includegraphics[width=0.85\linewidth]{../ExtFiles/MiTransCytInc.png}
            \caption{Multipass proteins.}
            \label{fig:MiTransCytInc}
        \end{subfigure}
        \caption{Translocation from the cytosol to the mitochondrial inner membrane.}
        \label{fig:MiTransCytIn}
    \end{figure}
    \begin{itemize}
        \item There are three methods by which this can occur.
        \item Method 1 (Figure \ref{fig:MiTransCytIna}).
        \begin{itemize}
            \item This method directly inserts single-pass transmembrane proteins into the inner membrane.
            \item Translocation begins as if the protein is to be pulled into the matrix, but immediately following the localization sequence, there is a stop-transfer sequence. When TIM interacts with this, it stops pulling the protein through, signal peptidase cleaves off the localization sequence, and TIM ejects the hydrophobic stop-transfer sequence into the inner membrane.
            \item Once TOM finishes pulling the bulk into the interluminal space and the protein refolds, we are done.
        \end{itemize}
        \item Method 2 (Figure \ref{fig:MiTransCytInb}).
        \begin{itemize}
            \item Use the OXA complex.
            \item The TIM/TOM complex moves a protein into the matrix and a single peptidase cleaves off the localization sequence.
            \item A secondary tag following the localization sequence then engages the OXA complex. The OXA complex flips the protein so that the tag is in the inner membrane and the bulk of the protein is in the interluminal space.
        \end{itemize}
        \item Method 3 (Figure \ref{fig:MiTransCytInc}).
        \begin{itemize}
            \item Multipass membrane proteins are introduced via the \textbf{TIM22 complex}.
        \end{itemize}
    \end{itemize}
    \item Import from the cytosol into the intermembrane space.
    \begin{figure}[H]
        \centering
        \begin{subfigure}[b]{\linewidth}
            \centering
            \includegraphics[width=0.15\linewidth]{../ExtFiles/MiTransCytLuma.png}
            \caption{Inner membrane cleavage.}
            \label{fig:MiTransCytLuma}
        \end{subfigure}
    \end{figure}
    \begin{figure}[h!]
        \ContinuedFloat
        \centering
        \begin{subfigure}[b]{\linewidth}
            \centering
            \includegraphics[width=0.38\linewidth]{../ExtFiles/MiTransCytLumb.png}
            \caption{Via Mia40.}
            \label{fig:MiTransCytLumb}
        \end{subfigure}
        \caption{Translocation from the cytosol to the mitochondrial interluminal space.}
        \label{fig:MiTransCytLum}
    \end{figure}
    \begin{itemize}
        \item If after insertion into the inner membrane, the bulk is cleaved from the transmembrane region, it will float away in the interluminal space. This is a secondary mechanism by which proteins enter the interluminal space, in addition to direct import by TOM.
        \item A third (and very popular) mechanism leverages disulfide bonds. These help the protein fold, but if the protein is to be pulled through TOM, these will have been reduced to split them and unfold the protein. When the reduced disulfide bonds interact with \textbf{Mia40} in the interluminal space, they get reassembled and Mia40 gets regenerated (it is a catalyst). This refolding sticks the protein in place.
    \end{itemize}
    \item Next time: Molecular mechanism of translocases; how they release transmembrane domains in the right origin.
    \item Peroxisomes.
    \begin{itemize}
        \item We understand very little about their function, but if anything is wrong with them, it's deadly.
        \item Take long lipid chains and cut them into shorter lipid chains by peroxidizing them (they have many reactive oxygen species).
        \item Smallest organelle in the cell (\SIrange{50}{100}{\nano\meter}) and has a very small number of proteins.
        \item Peroxisomes are thought to be born from the ER via budding. Then the peroxisome must mature and acquire proteins, both in its membrane and in the lumen.
        \item There are peroxisome targeting sequences, but who takes proteins to the peroxisomes and how they are transferred into the peroxisome is not clear.
        \item Peroxisomes are thought to undergo fission for replication, but we have no way to distinguish early peroxisomes from mature, functional ones.
        \item Peroxisomes carry their own catalase (which reduces oxygen into water and ROSs).
    \end{itemize}
    \item Summary of today.
    \begin{itemize}
        \item How compartments evolved; the evolution determines what is topologically equivalent to what. Nucleus and cytosol are equivalent (transport is facilitated by import and export factors), but most organelles are topologically equivalent to the extracellular matrix. You can take proteins to the nucleus or cytosol once they're born or to the mitochondria, or to specific places in the mitochondria.
    \end{itemize}
    \item Chemists are the best inventers, but they don't have a very good understanding of cell biology.
    \item GTPs are used for big conformational changes.
    \begin{itemize}
        \item The energy from hydrolyzing GTP and ATP is the same; it's just a question of how you use it molecularly.
    \end{itemize}
    \item Yamuna is very knowledgable about a variety of topics in her field.
\end{itemize}



\section{Quiz Prep}
\emph{From \textcite{bib:QuizReading}.}
\subsection*{Notes}
\begin{itemize}
    \item \marginnote{11/2:}\textbf{Single-stranded DNA}: DNA that is not currently bound and hydrogen bonded into a double helix. \emph{Also known as} \textbf{ssDNA}.
    \item \textbf{Transcription}: A highly dynamic process that generates ssDNA as transcription bubbles.
    \item \textbf{Transcription bubble}: A portion of the double helix that has been unwound and separated for the purpose of transcribing one strand of it.
    \item \textbf{KAS-seq}: Kethoxal-assisted single stranded DNA sequencing, the subject of this paper, provides rapid (within 5 mins), sensitive, and genome-wide capture and mapping of ssDNA produced by transcriptionally active RNA polymerases or other processes in situ using as few as 1000 cells.
    \item Kethoxal is a small molecule that rapidly and selectively binds with unpaired guanine.
    \item Attaching an azide group to kethoxal allows it to be tagged with bio-orthogonal click chemistry.
    \item Applications of KAS-seq.
    \begin{itemize}
        \item Definition of a group of single-stranded enhancers that enrich unique sequence motifs.
        \begin{itemize}
            \item Specifically, these enhancers are associated with the binding of specific transcription factors and exhibit elevated enhancer-promoter interactions.
        \end{itemize}
        \item Discovery: When \textbf{protein condensation} is inhibited, RNA polymerase II (Pol II) rapidly releases from a group of promoters.
        \item Fast and accurate analysis of transcription dynamics and enhancer activities simultaneously in both low-input and high-throughput modalities.
    \end{itemize}
    \item \textbf{Protein condensation}: Proteins sticking together.
    \item \textbf{Chromatin}: The material of which the chromosomes of organisms are composed, consisting of protein, RNA, and DNA.
    \item Transcription and its regulation (importance).
    \begin{itemize}
        \item Determine physiological function and the cell's fate.
        \item Regulation issues often lead to disease.
    \end{itemize}
    \item \textbf{Global transcription regulation}: Regulation of transcription across the entire genome.
    \item How do we understand global transcription regulation?
    \begin{itemize}
        \item Employ techniques like \textbf{ChIP-seq}.
        \item Search for the presence and level of \textbf{nascent RNA}.
        \begin{itemize}
            \item Based on \textbf{run-on assays}, \textbf{metabolic labeling}, and Pol II-associated or chromatin-associated RNA enrichment.
        \end{itemize}
    \end{itemize}
    \item \textbf{ChIP-seq}: A genome-wide sequencing approach that analyzes the occupancy of RNA polymerases.
    \item \textbf{Nascent RNA}: Newly made RNA, often still tethered to the DNA axis by elongating Pol II and being continuously altered by splicing and other processing events during its synthesis.
    \item \textbf{Assay}: An experimental method for assessing the presence, localization, or biological activity of a substance in living cells and biological matrices.
    \item \textbf{Run-on assay}: A method for measuring the frequency of transcription initiation. \emph{Also known as} \textbf{nuclear run-on assay}. \emph{Procedure}
    \begin{enumerate}
        \item Take cells. At the time you want to measure the frequency of transcription initiation, freeze them.
        \item Reheat them and incubate at \SI{37}{\celsius} in the presence of NTPs and radiolabeled UTP.
        \item Measure the amount of radiation given off by the products.
        \item The above measurement will be roughly proportional to the number of nascent transcripts on a gene at a certain time, which in turn is thought to be proportional to the frequency of transcription initiation.
    \end{enumerate}
    \item \textbf{Metabolic labeling}: The process of using the synthesis and modification machinery of living cells to incorporate detection or affinity tags into biomolecules.
    \item Limitations of the current methods for understanding global transcription regulation.
    \begin{itemize}
        \item Run-on assays and enrichment require millions of cells as starting material.
        \item Pol II ChIP-seq cannot distinguish whether RNA polymerases are simply bound or are actively engaged in transcription.
        \item Metabolic labeling cannot measure low-abundance RNA species. Post-transcriptional processing can also alter results.
    \end{itemize}
    \item If we want to understand global transcription regulation, then certainly it will be important to determine where transcription occurs.
    \item Goal: Locate where RNA polymerases engage in transcription.
    \begin{itemize}
        \item Observation: RNA polymerases transform dsDNA to ssDNA bubbles as they move.
        \item Method: Label/tag/identify/characterize ssDNA.
    \end{itemize}
    \item Previous attempts: \ce{MnO4-} preferentially oxidizes single-stranded thymidine residues.
    \begin{itemize}
        \item Has been used to reveal Pol II-induced promoter melting locally and on a genome-wide basis.
        \item Works together with S1 nuclease digestion.
        \item Doesn't work on B DNA.
        \item Limitations:
        \begin{itemize}
            \item Requires tens of millions of cells.
            \item Shows low sensitivity for weak/broad signals at Pol II elongation sites.
        \end{itemize}
    \end{itemize}
    \item Outline of the paper.
    \begin{itemize}
        \item Describe KAS-seq.
        \item Prove that it simultaneously measures the dynamics of transciptionally engaged Pol II, transcribing enhancers, Pol I and Pol III activities, and non-canonical DNA structures in which ssDNA plays a major role.
        \item Prove that it works with as few as 1000 cells.
        \item Prove that KAS-seq detects changes in transcription during quick environmental changes, e.g., inhibition of protein condensation.
    \end{itemize}
    \item Note that most conclusions listed here have supporting data and correlation numbers given in the paper.
    \item Genome-wide profiling of ssDNA using \ce{N3}-kethoxal-based labeling.
    \begin{figure}[h!]
        \centering
        \footnotesize
        \begin{subfigure}[b]{\linewidth}
            \centering
            \includegraphics[width=0.9\linewidth]{../ExtFiles/findssDNAa.png}
            \caption{\ce{N3}-kethoxal binding.}
            \label{fig:findssDNAa}
        \end{subfigure}\\[1em]
        \begin{subfigure}[b]{\linewidth}
            \centering
            \includegraphics[width=0.9\linewidth]{../ExtFiles/findssDNAb.png}
            \caption{Tagging and sequencing.}
            \label{fig:findssDNAb}
        \end{subfigure}
        \caption{Locating ssDNA within the genome.}
        \label{fig:findssDNA}
    \end{figure}
    \begin{itemize}
        \item Prior literature: Kethoxal reacts with the N1 and N2 positions of guanines (the ones that form Watson-Crick interactions) in ssDNA and RNAs under physiological conditions.
        \item This work: Attaching an azide "handle" to kethoxal to make \ce{N3}-kethoxal.
        \item Properties of \ce{N3}-kethoxal.
        \begin{itemize}
            \item Retains high activity and selectivity for guanine.
            \item Offers a bio-orthogonal handle that can readily be modified with a biotin or other FG.
        \end{itemize}
        \item \ce{N3}-kethoxal effectively maps the secondary structure of RNAs by selectively labeling guanines in ssRNAs under mild conditions in live cells.
        \item Hypothesis based on this result: The scope can be expanded to selectively labeling ssDNA (Figure \ref{fig:findssDNAa}).
        \item Verification of \ce{N3}-kethoxal's high labeling reactivity.
        \begin{itemize}
            \item Run an in vitro labeling assay using a synthetic DNA oligonucleotide with exactly four deoxyguanosine bases.
            \item \SI{37}{\celsius} and 5 mins incubation labels all deoxyguanosine bases.
        \end{itemize}
        \item Optimization of the KAS-seq \ce{N3}-kethoxal introduction conditions.
        \begin{itemize}
            \item Reaction occurs with deoxyguanosine within 2 mins.
            \item Reaction occurs with L-arginine within 10 mins.
            \item Thus, 5 mins is a good time to both tag deoxyguanosine and minimize protein labeling.
        \end{itemize}
        \item After labeling, \textbf{genomic DNA} is isolated and biotinylated through click chemistry.
        \item Enrich the fragments with \textbf{streptavidin} beads.
        \item Subject them to \textbf{library construction}.
        \item Remove \ce{N3}-kethoxal labels with a short heating at \SI{95}{\celsius}.
        \item Perform a PCR amplification.
        \item The whole process takes about 1 day.
    \end{itemize}
    \item \textbf{Genomic DNA}: Regular DNA inside the nucleus. \emph{Also known as} \textbf{gDNA}.
    \item \textbf{Streptavidin}: A protein with an extraordinarily strong binding affinity for biotin.
    \item \textbf{DNA library}: A collection of DNA fragments that have been cloned into vectors.
    \item \textbf{Library construction}: The act of storing and/or propagating a DNA library in a population of micro-organisms through the process of molecular cloning.
    \item Control experiments (run on one million live HEK293T cells\footnote{A derivative of a common strain of immortalized human kidney cells.} and mouse embryonic stem cells [mESCs]).
    \begin{itemize}
        \item Does \ce{N3}-kethoxal labeling affect gDNA isolation yield and purity? No.
        \item Do we still observe biotin signals in the absence of either \ce{N3}-kethoxal or the biotinylation reagent (biotin-DBCO)? No.
    \end{itemize}
    \item KAS-seq results are highly reproducible in replicate experiments.
    \item KAS-seq signals mark active transcription.
    \begin{itemize}
        \item KAS-seq signals exhibit a similar distribution pattern to Pol II ChIP-seq signals along regions with different G/C contents. Indicates G-specific labeling isn't a major factor.
        \item KAS-seq reads are very common at gene-coding regions, especially at gene promoters and transcription termination areas.
        \begin{itemize}
            \item On the other hand, they are far less common at intergenic regions.
        \end{itemize}
        \item KAS-seq signals positively correlate with known histone modifications denoting active transcription, and negatively correlate with inactive chromatin markers.
        \item KAS-seq also shows improvement over the permanganate method, particularly in the area of weak and broad ssDNA signals.
    \end{itemize}
    \item \textbf{Transition start site}. \emph{Also known as} \textbf{TSS}.
    \item \textbf{Transition end site}. \emph{Also known as} \textbf{TES}.
    \item KAS-seq works with very small numbers of cells.
    \begin{itemize}
        \item "Because of the high guanine labeling reactivity of \ce{N3}-kethoxal and the high affinity between biotin and streptavidin, KAS-seq is expected to maintain its sensitivity when using low-input starting materials or primary tissue samples" \parencite[516]{bib:QuizReading}.
        \item KAS-seq signals remain unchanged when using 10,000, 5,000, or even 1,000 HEK293T cells.
        \item KAS-seq retains its strong TSS signals but loses some of its gene body and TES signals when mouse liver tissue is used.
        \item Low-input of cells still yields similar enrichment efficiency.
    \end{itemize}
    \item KAS-seq reveals the dynamics of transciptionally engaged Pol II.
    \item Proof that what we're seeing is related to transciptionally engaged Pol II.
    \begin{itemize}
        \item KAS-seq results correlate well with results from \textbf{GRO-seq} and Pol II ChIP-seq.
        \item Experiments with inhibitors (DRB and triptolide) confirm that "the strong and sharp KAS-seq peaks on gene promoters reflect transcription initiation and pausing of Pol II near the TSS, and that KAS-seq signals at gene bodies are derived from transcription elongation" \parencite[517]{bib:QuizReading}.
        \item Treatment of the cells with DRB before performing KAS-seq decreased peak numbers by 57\% overall, primarily in the gene body and termination regions (signals went up at the TSS).
        \begin{itemize}
            \item This is the expected result, since DRB is known to inhibit Pol II release and keep it stuck at the TSS.
        \end{itemize}
        \item Treatment of the cells with triptolide before performing KAS-seq decreased peak numbers by 93\%.
        \begin{itemize}
            \item This is the expected result, since triptolide is known to inhibit Pol II being recruited to and loaded onto promoter regions.
        \end{itemize}
    \end{itemize}
    \item \textbf{GRO-seq}: Global run-on sequencing, which is the most widely used method to measure nascent RNA.
    \item What the dynamics of Pol II are.
    \begin{itemize}
        \item KAS-seq data from the promoter-proximal and gene body regions revealed that there are four classes of genes: Those for which Pol II pauses in the promoter or doesn't pause and those for which the gene is actively transcribed or isn't.
        \begin{itemize}
            \item This is consistent with previously reported GRO-seq studies.
        \end{itemize}
        \item KAS-seq data shows considerably enriched signals at the TES.
        \begin{itemize}
            \item DRB removes these, so they are from Pol II elongation and pausing at the end, not some attaching-in-a-different-place artifact.
            \item KAS-seq reads density on the terminal regions are all about the same, so KAS-seq doesn't exhibit length-dependent bias.
            \item KAS-seq gives a higher \textbf{termination index} than Pol II ChIP-seq and GRO-seq, suggesting Pol II accumulation at the TES is greater than previously expected.
        \end{itemize}
    \end{itemize}
    \item \textbf{Termination index}: The ratio of the reads density at the TES downstream regions relative to the density in the promoter-proximal regions.
    \item \textbf{RNA polymerase I}: The RNA polymerase that transcribes the 5.8S, 18S, and 28S rRNAs. \emph{Also known as} \textbf{Pol I}.
    \item \textbf{RNA polymerase III}: The RNA polymerase that transcribes the 5S rRNAs, tRNAs, and some small RNAs. \emph{Also known as} \textbf{Pol III}.
    \item KAS-seq detects Pol I- and Pol III-mediated transcription events and non-B form ssDNA structures in the same assay.
    \begin{itemize}
        \item Pol I- and Pol III-mediated transcription events are detected with Pol II ones as expected.
        \item These two do not respond to DRB or triptolide.
        \item Only about 2/3 tRNAs are actively transcribed, hinting at a transcription-level regulation of codon usage.
        \item Several KAS-seq peaks could not be paired to Pol I- or Pol III-mediated transcription events under DRB and triptolide conditions.
        \begin{itemize}
            \item Hypothesis: These could be from other DNA forms and telomeric DNA.
            \item Test: Used a previously reported method to predict where non-B form DNA species might exist in the genome and looked for overlaps with their mystery regions; found many.
            \item Takeaway: Using KAS-seq to study other ssDNA-involved biological processes could be a cool avenue to pursue in future research.
        \end{itemize}
    \end{itemize}
    \item Many enhancer regions are single-stranded, which correlates with higher enhancer activity.
    \begin{itemize}
        \item Since Pol II is known to bind at certain enhancers, we can use KAS-seq to identify enhancers that are being transcribed by Pol II.
        \item Identified \textbf{ssDNA-containing enhancers} under DRB conditions to focus on the TSS region.
        \item In mESCs, 25\% of enhancers are SSEs.
        \item Two SSE subtypes: KAS-seq signals span the whole enhancer, and the signals don't.
        \item SSEs include 94\% of super-enhancers.
        \item Genes associated with SSEs show higher expression levels.
        \item SSEs possess much more long-range interactions, indicating that these transcribing enhancers may possess a stronger capability to activate their target genes.
        \item SSEs enrich unique sequence motifs (??). Thus, they have distinct sequence features and transcription factor (TF) binding potentials.
        \item Comparison of SSEs and enhancers with high TF binding.
        \begin{itemize}
            \item ATAC-seq-positive enhancers are readily accessible.
            \item 50\% of these show no or very weak KAS-seq signals in mESCs.
            \item Genes associated with the KAS-seq-positive group show a higher expression level.
            \item There is a distinction between SSEs and motifs that are ATAC-seq-positive but KAS-seq-negative.
        \end{itemize}
        \item Pol II, histone modifications, and other transcription regulatory proteins are enriched on the SSEs.
        \item In HEK293T cells, the ratio of SSEs to general enhancers is lower, but all characteristics (overlap with super-enhancers, DRB response, and correlation with transcription regulatory proteins) are preserved.
        \item SSEs possess distinct genomic features and unique TF-binding footprints, as per our KAS-seq analysis.
    \end{itemize}
    \item \textbf{ssDNA-containing enhancer}. \emph{Also known as} \textbf{SSE}.
    \item \textbf{Protein condensate}: A highly dynamic structure formed through interactions between mediators, TFs, and other transcription coactivators that have been shown to incorporate Pol II to activate transcription.
    \item ssDNA dynamics upon the inhibition of protein condensates.
    \begin{itemize}
        \item 1,6-hexanediol dissociates protein condensates.
        \item The longer we let HEK293T cells sit in it, the more the KAS-seq signals diminish, supporting a role of protein condensate formation on transcription activation.
        \item Novel observation: After 5 mins, there is an increase in ssDNA clustered around the TSS.
        \begin{itemize}
            \item Leads to a slightly increased signal on the gene body and a coinciding decrease in signal at the TSS.
            \item The clusters form in both directions for bidirectionally transcribed genes and downstream, only, for unidirectionally transcribed genes.
            \item As time goes by, the clusters moved toward the TESs and gradually diminished.
        \end{itemize}
        \item Findings validated by Pol II ChIP-seq. KAS-seq even outdoes it in some places (e.g., detection of the above \textbf{fast-responsive genes}).
    \end{itemize}
    \item \textbf{Fast-responsive gene}: A gene with significant ssDNA cluster formation in the TSS region at 5 minutes.
\end{itemize}


\subsection*{Q \& A}
\begin{enumerate}
    \setitemize[1]{label={--}}
    \item The reason that KAS-Seq works on just 1000 cells as opposed to competing methods (e.g., ChIP-seq) that need millions of cells is:
    \begin{itemize}
        \item Kethoxal is highly reactive and specific to guanines.
        \item "Because of the high guanine labeling reactivity of \ce{N3}-kethoxal and the high affinity between biotin and streptavidin, KAS-seq is expected to maintain its sensitivity when using low-input starting materials or primary tissue samples" \parencite[516]{bib:QuizReading}.
    \end{itemize}
    \item How were the authors able to assign opened DNA structures to transcription and not replication?
    \begin{itemize}
        \item Experiments with inhibitors (DRB and triptolide) confirm that "the strong and sharp KAS-seq peaks on gene promoters reflect transcription initiation and pausing of Pol II near the TSS, and that KAS-seq signals at gene bodies are derived from transcription elongation" \parencite[517]{bib:QuizReading}.
    \end{itemize}
    \item Why do the authors incubate cells with kethoxal-\ce{N3} for such a short time (5 minutes) when incubation for a longer time will capture more ssDNA while the polymerase is transcribing?
    \begin{itemize}
        \item Kethoxal-\ce{N3}'s high binding affinity for guanines means that it will almost immediately attach to the target species, i.e., not more than 2 minutes is really needed. Additionally, given enough time (circa 10 minutes), it will begin to attach to other species, such as L-arginine (which also has two adjacent nitrogens). This leads to undesired tagging of proteins.
    \end{itemize}
    \item Which other purposes can kethoxal-\ce{N3} be used for?
    \begin{itemize}
        \item "Provides an effective way to map RNA secondary structures by labeling guanines in single-stranded RNAs under mild conditions in live cells" \parencite[515]{bib:QuizReading}.
    \end{itemize}
    \item How did the authors show that the background contribution upon subjecting cells to KAS-seq was negligible?
    \begin{itemize}
        \item "KAS-seq performed in the absence of \ce{N3}-kethoxal or the biotinylation reagent (biotin-DBCO) resulted in negligible biotin signals shown by dot blot, nor sufficiently enriched DNA for library construction, suggesting minimum background of KAS-seq" \parencite[516]{bib:QuizReading}.
    \end{itemize}
    \item Why do the authors use excess kethoxal with a short reaction time rather than a small amount of kethoxal incubated for a long time?
    \begin{itemize}
        \item See 3.
    \end{itemize}
    \item Glyoxal and methyl glyoxal are known cellular metabolites. Their accumulation is known to be disease causing. Can you explain how a disease might be caused?
    \begin{itemize}
        \item Inhibiting protein condensation leads Pol II to rapidly release from a group of promoters (as if the cell fears using up all of its energy when there might not be as much around).
        \item Likewise, perhaps it is possible that when there is too much protein, Pol II becomes hyperactive, consuming too much cellular energy, leading to oxidative stress and perhaps cell death.
        \item Lead to more reactive oxygen species, hence more oxidative stress.
    \end{itemize}
    \item What is the major advantage of a 5-minute kethoxal exposure, i.e., giving the cells a "pulse" of excess kethoxal that is then washed away?
    \begin{itemize}
        \item See 3.
    \end{itemize}
    \item KAS-Seq scores over ChIP-Seq in terms of its ability to work with frozen tissue samples. Why? Bear in mind that in frozen samples, the transcriptional bubbles remain.
    \begin{itemize}
        \item "Because of the high guanine labeling reactivity of \ce{N3}-kethoxal and the high affinity between biotin and streptavidin, KAS-seq is expected to maintain its sensitivity when using low-input starting materials or \emph{primary tissue samples}" \parencite[516]{bib:QuizReading}.
        \item Freezing denatures proteins, but does not alter transcriptional bubbles.
    \end{itemize}
    \item Which one(s) of the following descriptions is/are correct when we compare KAS-seq and ATAC-seq?
    \begin{itemize}
        \item "Notably, KAS-seq signals correlate better with H3K36me3 than ATAC-seq results do, indicating that while ATAC-seq serves as a powerful tool to probe chromatin accessibility, KAS-seq directly measures transcription activities" \parencite[516]{bib:QuizReading}.
        \item ATAC-seq is a tool to probe chromatin accessibility more broadly.
        \item Out of all ATAC-seq-positive enhancers, 50\% showed no (or very weak) KAS-seq signals. Thus, since KAS-seq is highly specific for ssDNA, this must mean that 50\% of ATAC-seq-positive enhancers are composed of dsDNA. Indeed, KAS-seq is more selective for ssDNA then ATAC-seq.
    \end{itemize}
\end{enumerate}


\subsection*{Proposed Answers}
\begin{itemize}
    \item 1-2, 2-1, 3-1, 4-(1),4, 5-4, 6-3, 7-3,(4), 8-5, 9-3, 10-4.
\end{itemize}




\end{document}